CMakeLists.txt

```diff
@@ -206,6 +206,9 @@ if (BUILTIN_EXTERNALS)
     if (BUILD_SNAPSHOTTER)
       set(ENV{BUILD_SNAPSHOTTER} "true")
     endif()
+    if (BUILD_CURL_WITHOUT_SSL)
```

Comment:
The BUILD_* arguments generally include/exclude parts of CVMFS in the build. I would rename it to something like NO_SSL_SUPPORT_EXPERIMENTAL to make sure that users know what they are doing when enabling this.